cfgov/jinja2/v1/events/_macros.html

```diff
@@ -70,11 +69,11 @@
    ========================================================================== #}
 
 {% macro event_meta_address(event, address_format='{city}, {state}') %}
-    {%- set city   =  event.venue_city     | default('Washington', true) -%}
-    {%- set state  =  event.venue_state    | default('DC', true) -%}
-    {%- set street =  event.venue_street   | default('1275 First St NE', true) -%}
-    {%- set name   =  event.venue_name     | default('CFPB', true) -%}
-    {%- set zip    =  event.venue_zip_code | default('20002', true) -%}
+    {%- set city   =  event.venue_city     | default('', true) -%}
+    {%- set state  =  event.venue_state    | default('', true) -%}
+    {%- set street =  event.venue_street   | default('', true) -%}
+    {%- set name   =  event.venue_name     | default('', true) -%}
+    {%- set zip    =  event.venue_zip      | default('', true) -%}
```

Comment:
Trivial: You could remove the leading spaces now that the first column is narrower.